pyvista/core/dataset.py

```diff
@@ -2019,6 +2019,9 @@ def _get_attrs(self):
         attrs = []
         attrs.append(("N Cells", self.GetNumberOfCells(), "{}"))
         attrs.append(("N Points", self.GetNumberOfPoints(), "{}"))
+        has_strips = getattr(self, "GetNumberOfStrips", None)
+        if callable(has_strips):
+            attrs.append(("N Strips", self.GetNumberOfStrips(), "{}"))
```

Comment:
I'd probably just check if it's an instance here with:
```suggestion
        if isinstance(self, pyvista.PolyData):
            attrs.append(("N Strips", self.n_strips, "{}"))
```

This is because it's guaranteed to have this API.